ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -992,6 +993,7 @@ private void setupTestState() throws InterruptedException, StoreException {
       // first log segment is filled to capacity.
       addCuratedData(segmentCapacity);
       assertEquals("Store size not as expected", expectedStoreSize, store.getSizeInBytes());
+      assertFalse("Expected nonempty store", store.isEmpty());
```

Comment:
this check is fine here but also required as soon as the first put occurs in `addCuratedData()` (to check that it returns `true` as soon as the first put arrives). 

Same with `CuratedLogIndexState`

Reply:
Done.